### DISCUSSION TITLE
vacuum.pause command

### DISCUSSION THREAD

**Discussion by matt1131** [humbertogontijo/homeassistant-roborock#192] [Q&A]
When I send the vacuum.pause command to my S7, it sets it ‘idle’ and is unable to resume whatever it was set to clean (e.g. segment, zone, etc.). Sending a vacuum.start command after the vacuum is either ‘idle’ or ‘paused’ causes it to start a full cleaning (even when paused from the app). Also, vacuum.start_pause throws an unknown error, regardless of what state the vacuum is in.

Is there a way to pause the current cleaning and then resume? Maybe by creating a new roborock.pause or roborock.start_pause command?

I’ll also add that after pausing from the app, I was able to resume cleaning from HA using

```
service: vacuum.send_command
data:
  command: resume_segment_clean
```

**Comment by Fuchschr11** [comment]
I have the same issue / problem.
